### PR TITLE
Create how-to-vex.mc

### PR DESCRIPTION
**Pull request by zvr**
copied over from https://github.com/spdx/spdx-spec/pull/933

### CODE REVIEW

**Comment by goneall** [comment]
Note that this was approved in the tech call for the other repo.

@rnjudge - if you're good with this, I can merge.

**Review by goneall — APPROVED** [review]
LGTM

File: docs/how-to-vex.mc [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+### J.1.1 Linking to VEX
+
+To link to an external VEX document, include an external reference of type `vulnerabilityExploitabilityAssessment` on the Vulnerability Element that encapsulates the CVE described in the VEX document.
+```json
+{
+  "@type": "Vulnerability",
+  "@id": "urn:spdx.dev:vuln-2",
+  "name": "cve-1234-5678",
+  "publishedTime": "2024-01-01T16:28:45Z",
+  "externalRefs": [
```
Comment (bact) [inline docs/how-to-vex.mc]:
```suggestion
  "externalRef": [
```
The property name has no trailing 's'.

**Review by bact — CHANGES_REQUESTED** [review]

File: docs/how-to-vex.mc (hunk 1 above)
Comment (bact) [reply docs/how-to-vex.mc]: Altenatively, we can merge this PR as it is first then fix this in another PR.

**Comment by kestewart** [comment]
@JPewdev - please review

**Review by bact — APPROVED** [review]

**Pull request merged**